charts/model-engine/values_circleci.yaml

```diff
@@ -96,6 +96,8 @@ config:
       profile_ml_worker: "default"
       profile_ml_inference_worker: "default"
     launch:
+      # Config to know where model-engine is running
+      gateway_namespace : default
```

Comment:
ideally we'd be able to not need to specify the namespace a second time here, since we pass that through the helm install command (e.g. `helm install ... --namespace <namespace>`) and can get the value through `{{.Release.namespace}}` (or whatever it is).
Having said that, getting that working also might be a bit ugly (e.g. we'd auto inject `gateway_namespace` into the config but nothing else)